CMakeLists.txt

```diff
@@ -91,3 +91,7 @@ macro (install_hook)
 endmacro (install_hook)
 
 include (OpmLibMain)
+
+add_test( NAME valgrind-restart
+          COMMAND /usr/bin/valgrind ${PROJECT_BINARY_DIR}/bin/test_Restart
```

Comment:
is hardcoding the path to valgrind a good idea? what happens if I compiled it myself and installed it into a custom location?